Formula/jack.rb

```diff
@@ -26,33 +22,18 @@ class Jack < Formula
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
+  depends_on "aften"
   depends_on "berkeley-db"
   depends_on "libsamplerate"
   depends_on "libsndfile"
 
-  on_linux do
-    depends_on "util-linux"
-  end
-
   def install
-    sdk = MacOS.sdk_path_if_needed ? MacOS.sdk_path : ""
-
-    # Makefile hardcodes Carbon header location
-    inreplace Dir["drivers/coreaudio/Makefile.{am,in}"],
-      "/System/Library/Frameworks/Carbon.framework/Headers/Carbon.h",
-      "#{sdk}/System/Library/Frameworks/Carbon.framework/Headers/Carbon.h"
-
-    # https://github.com/jackaudio/jack1/issues/81
-    inreplace "configure.ac", "-mmacosx-version-min=10.4",
-                              "-mmacosx-version-min=#{MacOS.version}"
-
-    system "./autogen.sh"
-    ENV["LINKFLAGS"] = ENV.ldflags
-    system "./configure", "--prefix=#{prefix}"
-    system "make", "install"
+    system "python", "./waf", "configure", "--prefix=#{prefix}"
```

Comment:
Which python should this be? Because right now it could be any python installed on the system. If this is the way to call the build script it should have a `depends_on "python@3.9"`.

Reply:
Just calling `system "./waf", etc.` makes the CI fail. Amended as requested.